packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testBrowseClassMethod.st

```diff
@@ -0,0 +1,16 @@
+tests
+testBrowseClassMethod
+	
+	| tally browser tree tallyForClassMethodCall |
+	
+	tally := IPTTreeModelMock constructMessageTallyFrom: [IPTTreeModelMock computeSomething].
```

Comment:
I could imagine that the test would be more robust if you used a `MessageTallyMock` here, as the assertion below is highly dependent on the implementation of the test itself.

Reply:
We agree with you in principle, but it seems to us that this is currently not easily possible with the current Mock. We note this down for further refactorings in the future!